src/Infrastructure/Persistence/PostgresUserRepository.hs

```diff
@@ -17,7 +17,9 @@ import Infrastructure.Persistence.Serializer (serializeUser, unserializeUser)
 import Tagger.EncryptedPassword (EncryptedPassword)
 import Tagger.Id (Id (Id))
 import Tagger.User (User (User))
-import Tagger.UserRepository (UserRepository (..))
+import Tagger.Repository.User (UserRepository (..))
+
+type Ctx = ExceptT UserRepositoryError IO
```

Comment:
I mentioned this also in the other PR. I don't consider such a type alias useful

Reply:
Yeah, this was a mistake. I removed it from the signatures but the declaration remained.